### ISSUE TITLE
[New App Shortcut]: Joyn

### ISSUE THREAD

**Issue by panteLx** [PRProd/HA-Firemote#234]
### App Name

Joyn

### Device Platform

Amazon Fire

**Comment by PRProd** [comment]
Greetings @panteLx!

I'm in the United States and my Fire TV device won't allow me to download an app by that name.  Also, I found a few different apps under that name while searching for APKs, so I'm not exactly sure which one you are requesting.

Please go to the [Details for App Requests](https://github.com/PRProd/HA-Firemote/wiki/How-Can-I-Help%3F#details-for-app-requests) page, follow the instructions [For Fire TV, Chromecast, NVIDIA Shield, Xiaomi Mi, and other Android based devices](https://github.com/PRProd/HA-Firemote/wiki/How-Can-I-Help%3F#for-fire-tv-chromecast-nvidia-shield-xiaomi-mi-and-other-android-based-devices), and post the result here.

Thanks.

**Comment by PRProd** [comment]
@panteLx - Have you had a chance to look at this yet?

**Comment by panteLx** [comment]
At the moment my whole server setup crashed and the restore won’t work as well 🥹 

I’ll come back to you asap.

**Comment by PRProd** [comment]
Ouch!  That sounds terrible!  Good luck on your restore / rebuild.

**Comment by PRProd** [comment]
@panteLx ~

I've been re-organizing (and hopefully improving) some of my documentation, so I wanted to give you the updated link to the [Providing Details for App Launcher Requests](https://github.com/PRProd/HA-Firemote/wiki/Providing-Details-for-App-Launcher-Requests) article:

https://github.com/PRProd/HA-Firemote/wiki/Providing-Details-for-App-Launcher-Requests#for-fire-tv-chromecast-nvidia-shield-xiaomi-mi-and-other-android-based-devices

Have you been able to recover from your crash yet?  If not, and if you think it might be a while, we could temporarily close this issue for now and re-open it when you are ready.  Just let me know either way.  Hopefully you have good news!

-Doug

**Comment by panteLx** [comment]
Hi,

ill send you all the requested details tomorrow. Got anything fixed so far. 

panteL

**Comment by panteLx** [comment]
```
adb_response: >-
  mSurface=Surface(name=de.prosiebensat1digital.seventv/de.prosiebensat1digital.seventv.MainActivity)/@0x55f520e
    mCurrentFocus=Window{b957ffe u0 de.prosiebensat1digital.seventv/de.prosiebensat1digital.seventv.MainActivity}
    mFocusedApp=AppWindowToken{b0da51e token=Token{61a5559 ActivityRecord{c8505a0 u0 de.prosiebensat1digital.seventv/.MainActivity t297}}}
```

**Comment by PRProd** [comment]
Thank you @panteLx!  

By any chance, do you speak German?  If so, would you be willing to help out a little bit more by clicking [this link](https://github.com/PRProd/HA-Firemote/issues/new?assignees=PRProd&labels=Region+Issue%2Cenhancement&projects=&template=New_Language_Support.yaml&title=%5BNew+Language+Support%5D%3A+German%20DE&langcode=de) and writing some translations while you wait for your app shortcut request?

**Comment by PRProd** [comment]
@panteLx ~

Since you are my favorite today, I decided to publish your changes right now.  Version [v3.1.2](https://github.com/PRProd/HA-Firemote/releases/tag/v3.1.2) is available for everyone now!

Enjoy!  Thank you once more for all of your help!

**Comment by ojherbst13** [comment]
Greetings @PRProd 

Could you also add Joyn to AppleTV?
https://apps.apple.com/at/app/joyn-%C3%B6sterreichs-superstreamer/id1239405046

**Comment by PRProd** [comment]
Greetings to you, @ojherbst13!

Absolutely!  Unfortunately that app is not available in my region, so I'm going to have to ask you to follow the [instructions listed here](https://github.com/PRProd/HA-Firemote/wiki/Providing-Details-for-App-Launcher-Requests#for-apple-tv-devices) and post your results before I can complete your request.

**Comment by ojherbst13** [comment]
### Application Details

source_list: Apple Events, App Store, Arcade, ARD Mediathek, Asphalt 8, Beach Buggy Racing, Computer, Das Erste, DAZN, Disney+, Einstellungen, Fibbage, Filme, Fitness, Fotos, Joyn, kabel eins, kabel eins Doku, Musik, Netflix, NFL, Plex, Podcasts, Prime Video, ProSieben, ProSieben MAXX, Protect, Ran, Real Racing 3, Red Bull TV, RTL+, SAT.1, SAT.1 Gold, ServusTV On, sixx, Sky X, Suchen, TV, TVthek, UFC, WELT Nachrichtensender, WOW, YouTube, ZDFmediathek

**Comment by PRProd** [comment]
Thanks for that detail @ojherbst13!

I've just published Firemote version [v3.3.0](https://github.com/PRProd/HA-Firemote/releases/tag/v3.3.0) which contains this request.  Enjoy!

-Doug
